config/eduro-subt.json

```diff
@@ -0,0 +1,104 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "app": {
+          "driver": "application",
+          "in": ["encoders", "scan", "emergency_stop", "pose2d", "buttons"],
+          "out": ["desired_speed", "pose2d", "artf_xyz"],
+          "init": {
+            "laser_pose2d": [0.13, 0.055, 0]
+          }
+      },
+      "fusion": {
+          "driver": "osgar.drivers.fusion:FusionPose2d",
```

Comment:
Why did you start using this long form specification in the config files? I've always meant to ask but forgot.

Reply:
Well the last time it was "OpenCV" camera and the reason was that not on all embedded devices is opencv installed. Here I was not sure if we want to use it as driver ... if you do not mind I would use "fusion" driver and modify `__init__.py`

Reply:
Can we have a more specific name? I think "fusion" is a way too generic. Why not include Pose2d in there as well? Similarly to the class name.

Reply:
do you mean "fusionPose2d"? Or "fusion_pose2d" or better "fusion_pose", because one of the outputs should be soon also xyz, i.e. 3D position for SubT artifacts